Rakefile

```diff
@@ -58,3 +58,22 @@ task(:rubocop) do
   cli = RuboCop::CLI.new
   cli.run(%w(-D -f s))
 end
+
+#### CHANGELOG ####
+begin
+  require 'github_changelog_generator/task'
+  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+    require 'puppet-strings/version'
+    config.future_release = "v#{PuppetStrings::VERSION}"
+    config.header = "# Changelog\n\n" \
+      "All significant changes to this repo will be summarized in this file.\n"
+    # config.include_labels = %w[enhancement bug]
```

Comment:
since we're pointing to the latest and greatest GCG, I'd love to use the `configure_sections` option to do the keepachangelog thing and then exclude maintenance stuff:
```ruby
config.configure_sections = {
      added: {
        prefix: "Added",
        labels: ["enhancement"]
      },
      fixed: {
        prefix: "Fixed",
        labels: ["bugfix"]
      },
      breaking: {
        prefix: "Changed",
        labels: ["backwards-incompatible"]
      }
    }
config.exclude_labels = ['maintenance']
```

Reply:
Added.